test/plots/barPlotTests.ts

```diff
@@ -775,6 +775,61 @@ describe("Plots", () => {
       });
     });
 
+    describe("Horizontal Bar Plot extent calculation", () => {
+
+      let svg: d3.Selection<void>;
+      let xScale: Plottable.Scales.Linear;
+      let yScale: Plottable.Scales.Linear;
+      let plot: Plottable.Plots.Bar<number, number>;
+
+      beforeEach(() => {
+        svg = TestMethods.generateSVG();
+
+        xScale = new Plottable.Scales.Linear();
+        yScale = new Plottable.Scales.Linear();
+
+        plot = new Plottable.Plots.Bar<number, number>("horizontal");
+        plot.x((d) => d.x, xScale);
+        plot.y((d) => d.y, yScale);
+      });
+
+      it("pads the domain in the correct direction", () => {
+
+        let data = Array.apply(null, Array(10)).map(function(d: any, i: any) {
+          return {
+            x: i + 1,
+            y: i
+          };
+        });
+        plot.addDataset(new Plottable.Dataset(data));
+        plot.renderTo(svg);
+
+        assert.operator(yScale.domain()[0], "<", data[0].y, "lower end of the domain is padded");
+        assert.operator(yScale.domain()[1], ">", data[data.length - 1].y, "higher end of the domain is padded");
+
+        svg.remove();
+      });
+
+      it("has the same size before and after autoDomain()-ing", () => {
```

Comment:
Would a more appropriate title be that calling autoDomain right after render does not change the domain?